contrib/kind.sh

```diff
@@ -583,6 +606,7 @@ check_ipv6
 set_cluster_cidr_ip_families
 create_kind_cluster
 docker_disable_ipv6
+docker_create_second_interface
```

Comment:
my bash is not great checks if that is correct
```suggestion
  if [ ! -z "$OVN_EX_GW_NETWORK_INTERFACE" ]; then
    docker_create_second_interface
  fi
```